guide/ops/server-cli-reference.md

```diff
@@ -74,6 +74,11 @@ This means that, once the task history is large, Brooklyn will continually use t
 memory. It will only expunge tasks from memory when this space is required for other objects within the
 Brooklyn process.
 
+### Web Console Bind Address
+
+To specify a local interface other than loopback (127.0.0.1) for the web console to bind to:
```

Comment:
The web console will by default bind to `0.0.0.0`. It's restricted to `127.0.0.1` only if `--noConsoleSecurity` is used.
The IP can also be set to `127.0.0.1` so suggest to change the wording here.